checker/check-response-property-one-off-updated.go

```diff
@@ -0,0 +1,111 @@
+package checker
+
+import (
+	"fmt"
+
+	"github.com/tufin/oasdiff/diff"
+)
+
+func ResponsePropertyOneOffUpdated(diffReport *diff.Diff, operationsSources *diff.OperationsSourcesMap, config Config) Changes {
```

Comment:
Typo: Off should be Of
Please rename to: ResponsePropertyOneOfUpdated

Reply:
thanks for spotting this, I pushed another commit fixing this typo (was in multiple places)